lib/presentation/journey/todo/todo_list/todo_list_screen.dart

```diff
@@ -54,6 +56,11 @@ class _TodoListScreenState extends State<TodoListScreen> {
                   case TodoFetchState:
                     return TodoListWidget(
                       state: state,
+                      onDismissable: (id) {
+                        _todoBloc.add(DeleteTodo(id));
+                        state.todos.data.removeWhere(
```

Comment:
Business logic such as removing an item should be done in BLOC.

Reply:
oh sure, you're right.
i should remove one of the list when the deletion from API is success, and i have to do it in bloc